azure-pipelines.yml

```diff
@@ -0,0 +1,88 @@
+# ASP.NET
+# Build and test ASP.NET projects.
+# Add steps that publish symbols, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/apps/aspnet/build-aspnet-4
+#trigger on master pull
```

Comment:
remove irrelevant comments from top of file
also, "master pull" should be "main push" if anything